docs/source/models/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -792,6 +792,60 @@ LDL-C decrease for an individual simulant can be calculated as:
 
 Where adherence score = 0 for primary or secondary nonadherent; and adherence score = 1 for adherent 
 
+
+Treatment Discontinuation
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In order to address overly optimisitic medication in the simulation, 
+we are adding a possibility for treatment discontinuation. The general 
+priniples for discontinuation are based on the observations of [An_2021]_ 
+which summarized the number of patients who discontinued treatment at 1 year 
+and 2 years after initialization. This approach can be summarized as: 
+
+- Approximately 31.4% of simulants who start medication will discontinue all treatments within 1 year (this is a weighted average across medication types)
+- Simulants who make it one year on treatment will continue on treatment indefinitely - this is based on the relatively low number of patients who discontinued treatment between year 1 and year 2 
+- Simulants who discontinue treatment will not restart medication at any point 
+
+In practice, these principles are implemented through rules applied to at initialization 
+and on time steps: 
+
+**On Initialization**
```

Comment:
Did you avoid using a burn-in approach due to computational constraints? Might want to note that here.